### PR TITLE
Add Sherwyne username devs.json

### PR DESCRIPTION
**Pull request by Sherwyne**

### CODE REVIEW

File: assets/data/devs.json [hunk 1]
```diff
@@ -120,7 +120,8 @@
   {
     "name": "Sherwyne Costiniano",
     "roles": ["Freelance Developer", "Flutter Developer", "DevOps"],
-    "about": "Python is my main programming language but I want to be versatile so I'll try Flutter and Dart"
+    "about": "Python is my main programming language but I want to be versatile so I'll try Flutter and Dart",
+    "username": "craSher"
```
Comment (Jansalvador1445) [inline assets/data/devs.json:124]: this should be `Sherwyne`, for the display image for the project :D

Comment (Sherwyne) [reply assets/data/devs.json:124]: I see, i'll commit another one :D

Comment (Jansalvador1445) [reply assets/data/devs.json:124]: Surething, Willing to wait.

**Comment by Jansalvador1445** [comment]
Merging this now.

**Pull request merged**